src/vs/workbench/api/browser/mainThreadEditors.ts

```diff
@@ -125,7 +125,7 @@ export class MainThreadTextEditors implements MainThreadTextEditorsShape {
 			// preserve pre 1.38 behaviour to not make group active when preserveFocus: true
 			// but make sure to restore the editor to fix https://github.com/microsoft/vscode/issues/79633
 			activation: options.preserveFocus ? EditorActivation.RESTORE : undefined,
-			ignoreOverrides: true
+			ignoreOverrides: uri.fsPath.endsWith('ipynb') || uri.fsPath.endsWith('sql') ? false : true // {{SQL CARBON EDIT}}
```

Comment:
do we need a toLower() here (i.e. a file is *.SQL or *.IPYNB)? Does that scenario work, or do we need a toLower() here?